wagtail/snippets/widgets.py

```diff
@@ -41,8 +42,9 @@ def render_js_init(self, id_, name, value):
                 app=model._meta.app_label,
                 model=model._meta.model_name)))
 
-    class Media:
-        js = [
+    @property
+    def media(self):
```

Comment:
When someone subclassed these, will this change break their code?

Can one use the `media` property method in the same as the `Media` class?